File: 002/002.js
const fibo = [0, 1]
for (let i = 2; i < 34; i++)
  fibo.push(fibo[i - 2] + fibo[i - 1])
console.log(fibo.filter(x => x % 2 === 0).reduce((a, b) => a + b, 0))

File: solutions/002/_002.cs
using System;

public class _002 {
    public static void Main() {
        int[] fibo = new int[36];
        fibo[0] = 0;
        fibo[1] = 1;
        for (int i = 2; i < 34; i++) {
            fibo[i] = fibo[i - 2] + fibo[i - 1];
        }
        int s = 0;
        for (int i = 0; i < 36; i++) {
            if (fibo[i] % 2 == 0)
                s += fibo[i];
        }
        Console.WriteLine(s);
    }
}
